WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js

```diff
@@ -2152,14 +2152,13 @@ function(findHits, includeInlineImages, includeInlineAtts) {
 
 			if (!this.isRealAttachment(attach) ||
 					(attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody && !includeInlineImages) ||
-					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.isRenderable(attach.contentType, true) && !includeInlineAtts) ||
+					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.textIsRenderable(attach.contentType, attach.isBody) && !includeInlineAtts) ||
```

Comment:
function name should be like isTextRenderable or isRenderableText ?

Reply:
Done